README.md

```diff
@@ -1,3 +1,7 @@
+Filer
```

Comment:
While you're touching the readme, use markdown for formatting. There are markdown editors online.